cfme/tests/containers/test_labels.py

```diff
@@ -0,0 +1,308 @@
+import json
+import random
+import re
+import time
+
+import fauxfactory
+
+import pytest
+
+import requests
+
+
+from cfme.containers.image import Image
+from cfme.containers.node import Node
+from cfme.containers.pod import Pod
+from cfme.containers.project import Project
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.replicator import Replicator
+from cfme.containers.service import Service
+from cfme.containers.template import Template
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable, toolbar as tb
+
+from utils import testgen, version
+from utils.appliance.implementations.ui import navigate_to
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: version.current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    [ContainersProvider], scope='function')
+
+
+TEST_OBJECTS = [Project, Service, Replicator, Pod, Node, Image, Template]
+
+objects_ocp = ['"Namespace"', '"Service"', '"ReplicationController"', '"Pod"',
+               '"Node"', '"Image"', '"Template"']
+
+
+def create_api_data(provider):
+
+    object_names = []
+    obj_keys_list = []
+    obj_values_list = []
+    obj_endpoints = []
+
+    # pick a random project name
+    # [0] is used to pick a name from the list, not necessarily the first item
+    # the project name will be random every time the test is executed
+
+    project_name = str(random.choice(provider.mgmt.list_project())[0])
```

Comment:
You forgot to commit some part of the code and this fails:
       return get_class_from_type(provider_data['type']).mgmt_class(**provider_kwargs)
E       TypeError: __init__() takes at least 2 arguments (2 given)